Cargo.toml

```diff
@@ -61,7 +61,7 @@ untrusted = "0.7.1"
 
 [dev-dependencies]
 base64 = "0.9.1"
-rcgen = { version = "=0.11.1", default-features = false }
+rcgen = { version = "0.11.1", default-features = false }
```

Comment:
Let's change this to `"0.11.3" since that would then ensure everybody avoids  0.11.2.

Reply:
Changed to specify 0.11.3